census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -231,6 +226,21 @@ private void recordMessageEvent(
     span.addMessageEvent(eventBuilder.build());
   }
 
+  private void recordAnnotation(
+      Span span, MessageEvent.Type type, int seqNo, boolean isCompressed, long size) {
+    StringBuilder messageTypeBuilder = new StringBuilder("compressed");
+    if (!isCompressed) {
+      messageTypeBuilder.insert(0, "un");
+    }
+    Map<String, AttributeValue> attributes = new HashMap<>();
+    attributes.put("id", AttributeValue.longAttributeValue(seqNo));
+    attributes.put("type", AttributeValue.stringAttributeValue(messageTypeBuilder.toString()));
+
+    String messageDirection = type.name().equalsIgnoreCase("Sent") ? "↗ " : "↘ ";
+    String inlineDescription = messageDirection + size + " bytes " + type.name().toLowerCase();
```

Comment:
`toLowerCase()` uses the system locale by default. Always pass an explicit locale to it, generally `Locale.ROOT` or `Locale.US` (which are generally interchangeable).

Reply:
TIL, updated.